content/blog/2022-07-FGBS1/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Fighting Galactic Bullshit 1"
```

Comment:
I think we should rebrand this a little. I'm not so sure about referring to the misconceptions people have as " bullshit". It feels a little aggressive and some people may perceive this as being blamed. I'm worried that we're annoying people for no reason, and it also seems a little desperate and makes it sound like we're struggling, which is far from the truth. 

I think it's better to create a positive context when promoting something instead of focusing on negative things. So instead of saying "we're here to correct the BS" maybe we can instead highlight the progress we've made since the start 15 years ago, which is gigantic! We can point out several features that have been added since some people first encountered galaxy, which they may not have had the chance to hear about. That should already go into the introductory text. This would give a solid framework for most of the blog posts to come.

Reply:
Most misconceptions are due to the fact that people simply do not know (similarly to how we sometimes do not know and make claims like "only synthetic oil is good for your engine"). But this is what bullshit is - normal oil is just fine if you do not have a turbo (or crazy compression).

Reply:
Bullshit implies (to some degree) that if you believe this, you've not been smart enough to understand. If you don't want to change the direction I'd be in favor of replacing Bullshit with the more boring `Misconceptions`.

Reply:
I think Bullshit refers to the arguments of people who don't care about true vs. false - just what is in their interest (see https://en.wikipedia.org/wiki/On_Bullshit).  I'm not sure that applies here.  I suspect that most people who hold/repeat some of the perceptions being discussed actually think they are true and care about what is true (so hopefully could be convinced otherwise).  Misconceptions seems like a serious and academic word to me - maybe best used later in the post. I wonder about something inspired by clickbait practicioners like "N surprising things that Galaxy is good for".   On the other hand - bullshit certainly catches the eye... we need some more marketing expertise!